lib/ansible/galaxy/role.py

```diff
@@ -365,3 +367,24 @@ def spec(self):
         }
         """
         return dict(scm=self.scm, src=self.src, version=self.version, name=self.name)
+
+    @property
+    def requirements(self):
+        """
+        Returns role requirements
+        """
+        if self._requirements is None:
+            self._requirements = []
+            for meta_requirements in self.META_REQUIREMENTS:
```

Comment:
this reads both files and overwrites with the 2nd, i would just break the loop once the first is read successfully

Reply:
Valid point. I based myself off https://github.com/ansible/ansible/blob/devel/lib/ansible/galaxy/role.py#L107-L118 though so it follows the same logic.

Reply:
I'm going to break on the first file found and keep throwing an exception if the parsing fails.